### PR TITLE
Feat/ Remove Invitation Mobile

### PR DESCRIPTION
**Pull request by desperado1802**
#1356
Added functionality for invitation to be removed
https://github.com/ever-co/ever-teams/assets/124465103/c52b0aa1-fb37-4a47-abe1-bec946b7ce95

### CODE REVIEW

File: apps/mobile/app/services/hooks/useTeamInvitation.ts [hunk 1]
```diff
@@ -68,6 +72,14 @@ export function useTeamInvitations() {
 		).then((res) => console.log(res))
 	}, [])
 
+	const removeSentInvitation = useCallback(async (inviteId: string) => {
+		await removeTeamInvitationsRequest({
+			invitationId: inviteId,
+			bearer_token: authToken,
+			tenantId,
+		}).catch((err) => console.log(err))
```
Comment (badalkhatri0924) [inline apps/mobile/app/services/hooks/useTeamInvitation.ts:80]: @desperado1802 Lets avoid console logs in mobile appliation.

File: apps/mobile/app/services/hooks/useTeamInvitation.ts [hunk 2]
```diff
@@ -68,6 +72,14 @@ export function useTeamInvitations() {
 		).then((res) => console.log(res))
 	}, [])
 
+	const removeSentInvitation = useCallback(async (inviteId: string) => {
+		await removeTeamInvitationsRequest({
```
Comment (badalkhatri0924) [inline apps/mobile/app/services/hooks/useTeamInvitation.ts:76]: @desperado1802 can we remove the card at the same time user remove the invite, updating local state and then do API call to remove?

We want to make the app very responsive. Even we will be doing that in web too gradually.

**Review by badalkhatri0924 — CHANGES_REQUESTED** [review]

File: apps/mobile/app/screens/Authenticated/TeamScreen/components/InviteCardItem.tsx [hunk 3]
```diff
@@ -179,7 +180,16 @@ const ListCardItem: React.FC<Props> = (props) => {
 										setShowMenu(!showMenu)
 									}}
 								>
-									Resend
+									{translate("tasksScreen.resendInvitation")}
+								</ListItem>
+								<ListItem
```
Comment (badalkhatri0924) [inline apps/mobile/app/screens/Authenticated/TeamScreen/components/InviteCardItem.tsx:185]: @desperado1802 let's take user confirmation too.

image

**Review by badalkhatri0924 — CHANGES_REQUESTED** [review]

File: apps/mobile/app/screens/Authenticated/TeamScreen/components/InviteCardItem.tsx (hunk 3 above)
Comment (desperado1802) [reply apps/mobile/app/screens/Authenticated/TeamScreen/components/InviteCardItem.tsx:195]: https://github.com/ever-co/ever-teams/assets/124465103/4ad9c64f-5f0c-4fc6-a199-cf247f57e608

**Review by badalkhatri0924 — APPROVED** [review]

**Pull request merged**